crypto/cms/cms_kemri.c

```diff
@@ -284,7 +283,7 @@ static int cms_kek_cipher(unsigned char **pout, size_t *poutlen,
     rv = 1;
 err:
     OPENSSL_free(out);
-    OPENSSL_cleanse(kek, keklen);
+    OPENSSL_cleanse(kek, sizeof(kek));
```

Comment:
I guess your change here is the best it can do for now..

Ideally it would not even goto err for the first if.. But that would probably still require
setting
````
    EVP_CIPHER_CTX_reset(kemri->ctx);
    EVP_PKEY_CTX_free(kemri->pctx);
    kemri->pctx = NULL;
````
(It is fairly hard to follow the flow of the CMS code, when the creation and frees are all over the place).